DoSomething/phoenix-next
# Display school impact in School Finder

### What's this PR do?

This pull request adds support for an additional, optional SchoolFinder parameter called `actionId`, used to display action stats for the user's selected school.

~~This PR is draft -- still needs final styles, and could use additional Cypress tests.~~ -- done

### How should this be reviewed?

* Select a school
* Submit a photo post
* Review the photo post
* Refresh campaign, verify total has updated 

### Any background context you want to provide?

For now these SchoolFinder parameters are configured via `additionalContent` of a `SectionBlock`, but it'd be ideal to move this into a `SchoolFinderBlock` content type in a future PR if we plan on re-using School Finders long-term.

### Relevant tickets

References [Pivotal #169659230](https://www.pivotaltracker.com/n/projects/2401401/stories/169659230).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Added screenshots of front-end changes on small, medium, and large screens.
- [x] Added appropriate feature/unit tests.
